Final_Project/Main.cpp

```diff
@@ -97,6 +99,18 @@ int WINAPI WinMain(HINSTANCE hInst, HINSTANCE hprevInstance,
 	//set LAST_EOT_RECEIVED to current time
 	LAST_EOT_RECEIVED = time(0);
 
+	//open com port
+	if ((portHandle = CreateFile(lpszCommName, GENERIC_READ | GENERIC_WRITE, 0,
+		NULL, OPEN_EXISTING, 0, NULL))
+		== INVALID_HANDLE_VALUE)
+	{
+		MessageBox(NULL, TEXT("Error opening COM port:"), TEXT(""), MB_OK);
+		return FALSE;
```

Comment:
I don't think there's any need to return false here. Maybe we can exit the program?

Reply:
that would make sense